### PR TITLE
Added cherries.

### PR DESCRIPTION
**Pull request by jellygato**
## About the PR
Added cherries and cherry seeds.

## Why / Balance
Part of issue #302 . A lot of recipes from Paradise rely on cherry jelly, which is hard to make with no cherries.

## Technical details
Added cherry YAML and cherry seed YAML, along with localization files. Created Goobstation-specific files and folders where possible (which I believe was everywhere except the seed vending machine inventory)  to keep code easily discernible from WizDen code. No C# changes were made. 

Cherries have the same growth time and nutrition as apples, but cannot be mutated. Harvesting them will grant you three cherries. You can use them in the seed extractor to get more cherry seeds. Right now their only culinary use is for juicing; recipes using cherries will be added in the near future.

## Media

Cherry and packet of cherry seeds. Cherry sprite came from Paradise Station. Cherry seeds sprite I made using the apple seeds packet as a template.
image

A growing cherry tree.
image

Cherries are burgerable and tacoable.
image

Cherry juice is red.
image

## Requirements

- [x ] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Cherries have been added, as well as the ability to grow your own cherries!

## Summary by CodeRabbit

- **New Features**
	- Added new localization entries for cherry-related items, enhancing user experience with localized content for flavors, food sequences, and consumables.
	- Introduced cherry seeds and a new cherry entity in the hydroponics system, expanding agricultural options for players.
	- Added metadata for cherry plant growth stages to improve gameplay mechanics.

- **Bug Fixes**
	- Addressed issue #302, improving the inventory options for cherry seeds in vending machines.

### CODE REVIEW

**Comment by jellygato** [comment]
Also if you have a better idea for cherry flavortext other than "The best red flavor" please go ahead and update it. I couldn't think of anything witty.

**Comment by Piras314** [comment]
CL doesn't fit the format

should be like
```
:cl:
- add: Cherries have been added along with some corresponding foodstuffs!
```

File: Resources/Textures/Goobstation/Objects/Specific/Hydroponics/cherry.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+  {
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/ParadiseSS13/Paradise/commit/bd91a1b962fe9bd38e346e9fafd4ebb10784fcb3",
```
Comment (Piras314) [inline Resources/Textures/Goobstation/Objects/Specific/Hydroponics/cherry.rsi/meta.json:4]: "All but seed.png taken from (paradise link), seed.png made by jellygato"

**Review by Piras314 — CHANGES_REQUESTED** [review]
1 copyright change, surprisingly good other than that

**Comment by jellygato** [comment]
I've updated the copyright in the meta.json with your recommended verbiage -- thanks! The change was made to the file directly here in Github to the same branch. I'm still getting used to the ins and outs of Github so please let me know if you need me to resubmit the pull request so it lassos in the new meta.json.

I will also update the CL in the pull request.

**Comment by jellygato** [comment]
OLD CL: 

:cl: - Added cherries and cherry seeds.

NEW CL:

:cl:
- add: Cherries have been added, as well as the ability to grow your own cherries!

**Review by Piras314 — APPROVED** [review]

**Comment by Piras314** [comment]
great job

review @whateverusername0

**Comment by whateverusername0** [comment]
looks good ig

**Pull request merged**